conf/messages.cy

```diff
@@ -877,6 +883,13 @@ addSecondOtherOfficials.p3 = (W)Adding information about other officials takes a
 addedSecondOtherOfficial.checkYourAnswers.title = (W)Check the second other official''s details(W)
 addedSecondOtherOfficial.checkYourAnswers.heading = (W)Check the second other official''s details(W)
 
+# Charity's Other Officials Details Summary Page Messages
+# ----------------------------------------------------------
+otherOfficialsSummary.checkYourAnswers.title = (W)Check the charity''s other official details(W)
```

Comment:
please check page title and heading, content should be from ticket

Reply:
This is accurate to the prototype page

Reply:
It was mentioned that we should follow tickets and then will get additional ticket for content changes